packages/fxa-settings/src/lib/storage-utils.ts

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Storage from './storage';
+
+export function isOriginalTab() {
+  const storage = Storage.factory('sessionStorage');
```

Comment:
I'm not sure if we need a `storage` vs `storage-data` (in the model-data) ticket since we're doing a lot of the same here, but I was torn between placing these methods on the integrations and referencing `StorageData` on them vs doing this vs creating a hook to pull this off `AppContext`.